### PR TITLE
Update release instructions to publish artifacts for different scala versions

### PR DESCRIPTION
**Pull request by vjagadish1989**

### CODE REVIEW

**Comment by vjagadish1989** [comment]
@jmakes for review

File: RELEASE.md [hunk 1]
```diff
@@ -121,6 +121,14 @@ In order to ensure that the release is available in public mirrors, wait for the
 to show up in [maven central](http://search.maven.org/#search%7Cga%7C1%7Csamza). A full list of mirrors can be found [here](http://www.apache.org/dyn/closer.cgi).
 Do not publish the website or any public document until the release jars are available for download.
 
-## Steps to Update Public Document
+## Steps to Publish Artifacts for other Scala Versions
+The above instructions publish the artifacts for scala 2.11. To publish for scala 2.10 and 2.12:
```
Comment (prateekm) [inline RELEASE.md]: This isn't optional, so this should be part of "Steps to publish binary artifacts" instead of separate section.

"instructions above".

Comment (vjagadish1989) [inline RELEASE.md]: Fixed in the updated draft. thanks! 
Did it come across as "optional" because it was in a separate section?

**Comment by vjagadish1989** [comment]
Address @prateekm 's review feedback.

**Pull request closed without merging**